flask_server.py

```diff
@@ -4,16 +4,20 @@
 import base64
 import shutil
 import settings
+import logging
 
 app = Flask(__name__)
 app.config['TEMPLATES_AUTO_RELOAD'] = True
 
 SKIP_EXTENSIONS = (".json", ".txt")
 
+settings.setup_logger()
+logger = logging.getLogger(__name__)
+
 ################### UTIL FUNCTIONS ###################
 
 def get_log_file_list():
-    with open(settings.get('log_file_path'), "r+") as o:
+    with open(settings.get('scraping_log_file_path'), "r+") as o:
```

Comment:
Leave the name as log_file_path, we will use the logger for also other purposes.